server/graphql/v1/queries.js

```diff
@@ -931,6 +931,10 @@ const queries = {
         description: 'Whether to skip recent accounts (48h)',
         defaultValue: false,
       },
+      countries: {
+        type: new GraphQLList(GraphQLString),
```

Comment:
Why not use an Enum instead of a generic GraphQLString?

Reply:
Thanks, that makes sense, we already have the `CountryISO` enum, so I've used that. 👍🏽

Reply:
According to https://github.com/opencollective/opencollective/issues/4378, shouldn't that be `country`?